src/styles/main.scss

```diff
@@ -1,3 +1,27 @@
+@import url(./blocks/nav.scss);
+@import url(./blocks/cards.scss);
+@import url(./blocks/stars.scss);
+
+:root {
+  --card-main: #060b35;
+  --card-fallback: #616070;
+  --card-secondary: #00acdc;
+  --card-border: #f3f3f3;
+}
+
+html {
+  font-size: 12px;
+  font-family: "Roboto", sans-serif;
+}
+
 body {
   margin: 0;
 }
+
+.header {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  padding: 0 50px;
+  box-shadow: 0 2px 4px rgba(0, 0, 0, 0.05);
+}
```

Comment:
for such a small project i'd rather agree to keep these
but for bigger projects try to have only imports inside `main.scss` file
and list in separate files your
- variables
- normalization
- resets
- etc